Dockerfile

```diff
@@ -1,6 +1,12 @@
-FROM golang:1.13.4-buster
+FROM golang:1.13.6-buster
 LABEL maintainer="Steven Allen <steven@stebalien.com>"
 
+# Install deps
+RUN apt-get update && apt-get install -y \
```

Comment:
Note: I think moving this to the first line means we'll have to explicitly run with `--no-cache=true` when we want to re-update those dependencies. The layer cache uses the string value of the RUN command to see if already has a layer for that.

> aside from the ADD and COPY commands, cache checking does not look at the files in the container to determine a cache match. For example, when processing a `RUN apt-get -y update` command the files updated in the container are not examined to determine if a cache hit exists. In that case just the command string itself is used to find a match.

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#leverage-build-cache

Reply:
That's probably for the best, actually. When rebuilding, we usually just want to update ipfs itself.